common/authentication/postgresql/metadata.go

```diff
@@ -91,15 +91,15 @@ func (m *PostgresAuthMetadata) ValidateAwsIamFields() (string, string, string, e
 	if awsRegion == "" {
 		return "", "", "", errors.New("metadata property AWSRegion is missing")
 	}
-	awsAccessKey, _ := metadata.GetMetadataProperty(m.awsEnv.Metadata, "AWSAccessKey")
-	if awsAccessKey == "" {
+	accessKey, _ := metadata.GetMetadataProperty(m.awsEnv.Metadata, "AccessKey")
```

Comment:
Is this a breaking change?

Reply:
if we wish keep the same name with the `aws` prefix i can update it 👍

Reply:
and i notice this `ValidateAwsIamFields()` function is only used for postgresql, all the other components are already migrated to use the built-in aws profile instead of defining them in the metadata.yaml.  Shall we keep this function or make any tweak on it? Would you give me some guidance please @ItalyPaleAle ?

Reply:
We need to retain backwards compatibility

Reply:
agree, changed to another logic to achieve the same goal: for now just skip append duplicated aws fields during the build process. I believe this can keep the naming and previous AWSIAM validation logic, while drop the duplicated fields :)